text/0000-fabric-website.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: Fabric_Website
+nav_order: 3
+---
+
+- Feature Name: Fabric Website
+- Start Date: (fill me in with today's  date, 2020-08-14)
+- RFC PR: (leave this empty)
+- Fabric Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This RFC details the proposal for a new website for using Hyperledger Fabric. This website would serve as a "front door" for the project, and collate the various resources available to help users get started with Fabric.
+
+The website has been deployed to Heroku for development purposes and is available to browse [here](https://warm-reaches-04614.herokuapp.com/).
```

Comment:
The link page loading time on my first access via browser is more than I expect. A quick analysis shows the pending of wss://warm-reaches-04614.herokuapp.com/sockjs-node last for around 50 seconds.

Reply:
That's because Heroku (i guess it's free tier?) puts inactive applications to sleep, and wakes them up when traffic is incoming. It has nothing to do with the site itself, I think.

Reply:
Yes that's right, we're hosting on the free version of Heroku so the load time will be quite long if the site hasn't been active in a while. We won't see this when we deploy to gh-pages.